episodes/07-infrastructure.md

```diff
@@ -119,6 +119,12 @@ you should be presented with a brand new lesson repository.
 
 TODO insert screenshot of repository structure here
 
+::: instructor
+
+Remind learner to paste URL of their lesson repo to the collaborative document.
```

Comment:
Do you think it would be a good idea to also ask them to share the URL of the rendered lesson pages, as well as the repository?

Reply:
Yes, good point I'll add that.

Reply:
That is ready now. Can you check again?